services/ui-src/src/utils/other/export.tsx

```diff
@@ -93,6 +93,16 @@ export const renderDrawerDataCell = (
       parentFieldCheckedChoiceIds &&
       !parentFieldCheckedChoiceIds?.includes(entity.id);
     const fieldResponseData = entity[formField.id];
+
+    // check for nested ILOS data
+    let nestedIlosResponses;
+    if (
+      fieldResponseData?.length &&
+      formField.id === "plan_ilosUtilizationByPlan"
+    ) {
+      nestedIlosResponses = getNestedIlosResponses(fieldResponseData, entity);
```

Comment:
`renderDrawerDataCell` doesn't have its own tests and it does seem tricky to test. Here's a suggestion to test this scenario:
```
test("renderDrawerDataCell renders ilos responses", () => {
  const mockForm: FormJson = {
    id: "plan_ilosUtilizationByPlan",
    fields: [
      {
        id: "mock-field-id",
        props: {
          choices: [
            {
              id: "mock-choice-id-1",
              label: "mock label 1",
            },
            {
              id: "mock-choice-id-2",
              label: "mock label 2",
            },
          ],
        },
        type: "radio",
        validation: "radio",
      },
    ],
  };

  const mockPlan: AnyObject = [
    {
      id: "mock-id",
      plan_ilosUtilizationByPlan: [
        {
          key: "123",
          value: "mock-ilos",
        },
      ],
      plan_ilosUtilizationByPlan_123: "N/A",
    }      
  ]

  const cells = renderDrawerDataCell(mockForm, mockPlan, "drawer");
  const ul = cells[0].props.children;
  const li = ul.props.children
  expect(cells.length).toBe(1);
  expect(li.length).toBe(3);
});
```

Reply:
`expect(li.length).toBe(3);` is actually giving a false positive.

Reply:
I think this is out of the scope for this one, but I'll create a ticket for increasing test coverage and make a note of this example! thank you, @bangbay-bluetiger 🥇 great call.

Reply:
@bangbay-bluetiger that's super helpful, I included this unit test in the PR. thank you!!